test/mpi/main.cc

```diff
@@ -59,29 +61,45 @@ static std::vector<std::string> str_split(const char *value, const char *delimit
 
 void PrintHelp()
 {
-    std::cout <<
```

Comment:
i would roll back this hunk of change. i understand it comes from clang-format but imho it makes the output more difficult to read